src/alda/Main.java

```diff
@@ -179,6 +183,7 @@ public static void main(String[] argv) {
     jc.addCommand("version", version);
 
     jc.addCommand("play", play);
+    jc.addCommand("stop", stop, "stop-playback");
```

Comment:
That's an interesting idea, but I think I like the `alda play` / `alda stop` / `alda pause` scheme better. Right now the list of CLI commands available is just a flat map where `alda foo` does `bar`.

In the future as Alda grows more complicated, we might consider a more organized scheme where `alda foo bar` does `baz`, but right now I don't see the need. It's something we can always do in the future and we could bump the major version of Alda so that it's clear that we're making a breaking change.